docs/fenix.md

```diff
@@ -2,50 +2,57 @@
+Android Studio provides virtual devices _via_ the [AVD Manager](https://developer.android.com/studio/run/managing-avds) in case you do not have a real Android-compatible device. In order to run the Fenix debug build created before, either configure a new virtual device or use your own device, then run the following command in the Fenix directory (inside the `firefox-android` repository):
 
 ```
-$ ./gradlew installDebug
+$ cd /path/to/firefox-android/fenix/
+$ ./gradlew installFenixDebug
 ```
 
 For more information, you can find more information on this page: https://developer.android.com/studio/build/building-cmdline#RunningOnDevice
 
 ## Custom build for AMO
 
-Once you ensured that you can compile, install and run a debug Fenix build, let's update the build configuration for AMO -stage. Only a few values have to be changed in the `app/build.gradle` file as shown below:
+Once you ensured that you can compile, install and run a debug Fenix build, let's update the build configuration for AMO -stage. Only a few values have to be changed in the `/path/to/firefox-android/fenix/app/build.gradle` file as shown below:
 
 ```diff
-
-diff --git a/app/build.gradle b/app/build.gradle
-index cd7c44f2d..78e0ffd2e 100644
---- a/app/build.gradle
-+++ b/app/build.gradle
-@@ -38,11 +38,11 @@ android {
-         resValue "bool", "IS_DEBUG", "false"
+diff --git a/fenix/app/build.gradle b/fenix/app/build.gradle
+index 81d60db1fc..f3c4d53ebd 100644
+--- a/fenix/app/build.gradle
++++ b/fenix/app/build.gradle
+@@ -45,9 +45,9 @@ android {
          buildConfigField "boolean", "USE_RELEASE_VERSIONING", "false"
+         buildConfigField "String", "GIT_HASH", "\"\"" // see override in release builds for why it's blank.
          // This should be the "public" base URL of AMO.
 -        buildConfigField "String", "AMO_BASE_URL", "\"https://addons.mozilla.org\""
--        buildConfigField "String", "AMO_COLLECTION_NAME", "\"7dfae8669acc4312a65e8ba5553036\""
+-        buildConfigField "String", "AMO_COLLECTION_NAME", "\"Extensions-for-Android\""
 -        buildConfigField "String", "AMO_COLLECTION_USER", "\"mozilla\""
 +        buildConfigField "String", "AMO_BASE_URL", "\"https://addons.allizom.org\""
 +        buildConfigField "String", "AMO_COLLECTION_NAME", "\"fenix\""
 +        buildConfigField "String", "AMO_COLLECTION_USER", "\"11686491\""
+         // These add-ons should be excluded for Mozilla Online builds.
+         buildConfigField "String[]", "MOZILLA_ONLINE_ADDON_EXCLUSIONS",
+                 "{" +
+@@ -62,7 +62,7 @@ android {
+                         "\"adnauseam@rednoise.org\"" +
+                 "}"
          // This should be the base URL used to call the AMO API.
 -        buildConfigField "String", "AMO_SERVER_URL", "\"https://services.addons.mozilla.org\""
 +        buildConfigField "String", "AMO_SERVER_URL", "\"https://addons.allizom.org\""
```

Comment:
For consistency since we do have that set up:

```suggestion
+        buildConfigField "String", "AMO_SERVER_URL", "\"https://services.addons.allizom.org\""
```